daft/dataframe/dataframe.py

```diff
@@ -154,39 +154,6 @@ def columns(self) -> List[Expression]:
         """
         return [col(field.name) for field in self.__builder.schema()]
 
-    @DataframePublicAPI
-    def show(self, n: int = 8) -> "DataFrameDisplay":
-        """Executes enough of the DataFrame in order to display the first ``n`` rows
-
-        .. NOTE::
-            This call is **blocking** and will execute the DataFrame when called
-
-        Args:
-            n: number of rows to show. Defaults to 8.
-
-        Returns:
-            DataFrameDisplay: object that has a rich tabular display
-        """
-        builder = self._builder.limit(n, eager=True)
-
-        # Iteratively retrieve partitions until enough data has been materialized
-        tables = []
-        seen = 0
-        for table in get_context().runner().run_iter_tables(builder, results_buffer_size=1):
-            tables.append(table)
-            seen += len(table)
-            if seen >= n:
-                break
-
-        preview_partition = Table.concat(tables)
-        preview_partition = preview_partition if len(preview_partition) <= n else preview_partition.slice(0, n)
-        preview = DataFramePreview(
-            preview_partition=preview_partition,
-            # We do not know the size of the entire (un-limited) dataframe when showing
-            dataframe_num_rows=None,
-        )
-        return DataFrameDisplay(preview, self.schema(), num_rows=n)
-
```

Comment:
Moved `df.show()` to be defined next to `df.collect()` since them being far away from each other was a bit annoying. 😛